src/main/java/com/sigopt/model/Metric.java

```diff
@@ -10,6 +10,11 @@ public Metric(String name) {
         this.set("name", name);
     }
 
+    public Metric(String name, String objective) {
+        super();
+        this.set("objective", objective);
+    }
+
```

Comment:
hmm do we need this? Looks like we don't have one for `value_baseline`

Reply:
Good point, thank you will remove!